browses.md

```diff
@@ -54,6 +54,39 @@ Error codes:
 
 **404** if the browse index doesn't exist
 
+## Hierarchical browse index 
+**/api/discover/browses/<:index-name>**
+
+Provide detailed information about a specific hierarchical browse index and access to the parameters required to access the list of items and entries. The JSON response document is as follows
+```json
+{
+  "id": "keyword",
+  "facetType": "subject",
+  "vocabulary": "srsc",
+  "type": "hierarchicalBrowse",
```

Comment:
we cannot use the same REST collection endpoint to expose different types of resources. We can eventually replace the metadataBrowse flag with a browseType attribute